### DISCUSSION TITLE
Bootstrap cdn Reference

### DISCUSSION THREAD

**Discussion by Jireck-npds** [npds/npds_dune#517] [Ideas]
Bonjour,
Pourquoi ne pas mettre une liaison CDN Bootstrap, ou vers le bootstrap du site ?

https://github.com/npds/npds_dune/blob/f4a84eae182cccc84489a32a9054aab4de667846/revolution_16/mainfile.php#L435

**Comment by jipexu** [comment]
oh yes c'est une très bonne idée !!! (ça fera un mail plus lourd et notre bilan carbonne !???  ;)
je note ! et je ferai des essai pour cette thémisation des email en html

**Reply by Jireck-npds** [reply]
ca ferait des mails plus beau il me semble et plus simple car send_mail et certains choses ne sont pas optimisé

**Reply by Jireck-npds** [reply]
>  c'est une très bonne idée !!

J'en ai parfois
